src/test/java/org/folio/rest/MockServer.java

```diff
@@ -183,17 +183,12 @@ private void handleGetInstanceRecord(RoutingContext ctx) {
   private void handleGetSRSRecord(RoutingContext ctx) {
     logger.info("handleGetSRSRecord got: " + ctx.request()
       .path());
-    String query = ctx.request()
-      .getParam("query");
     try {
-      JsonObject srsRecords;
-      srsRecords = new JsonObject(RestVerticleTestBase.getMockData(SRS_RECORDS_MOCK_DATA_PATH));
+      JsonObject srsRecords = new JsonObject(RestVerticleTestBase.getMockData(SRS_RECORDS_MOCK_DATA_PATH));
       //fetch the ids from the query and remove them from the mock if not in the request
-      List<String> instanceIds = Arrays.stream(query.split("or"))
-        .map(st -> st.split("==")[1].trim().replace(")", ""))
-        .collect(Collectors.toList());
+      List<String> instanceIds = ctx.getBodyAsJsonArray().getList();
```

Comment:
I think the part of instanceIds filtering is still needed, That is how integration tests are run based on the request.

Reply:
The filtering is still in place, I just take instanceIds from request body, not from request url